src/internal/translations/en-US.json

```diff
@@ -164,6 +164,31 @@
         "comment": "",
         "limit": 0
     },
+    "dashboard.bucket.location_title.pushpin": {
+        "value": "Location",
+        "comment": "",
```

Comment:
Please provide the context. Sth like "Geological location (position of a pin pushed to a map)".